lib/rules/custom-property-pattern/__tests__/index.js

```diff
@@ -125,3 +125,18 @@ testRule({
 		},
 	],
 });
+
+testRule({
+	ruleName,
+	customSyntax: 'postcss-scss',
+	config: ['$foo-color'],
```

Comment:
```suggestion
	config: [/foo-.+/],
```

Let's use the same config as above as using a separate `testRule` is only about:
- keeping non-standard syntax away from standard syntax
- ensuring a custom syntax is used (even if the default parser can parse the construct)